client/src/components/game/lab5/pages/GameEnd.js

```diff
@@ -0,0 +1,60 @@
+import React, { Component, Fragment } from "react";
+import { navigate } from "@reach/router";
+import UserLabService from "../../../../services/UserLabService";
+import {LAB_ID} from "../../../../constants/lab5";
+import {Forum, EmojiObjects, Timer} from "@material-ui/icons";
+
+class GameEnd extends Component {
+  handleHome() {
+    navigate("/Lab5/Game/GameStart");
+  }
+
+  componentDidMount(){
+    UserLabService.complete_game(LAB_ID);
+  }
+  render() {
+    // const { user, state, plays } = this.props;
+    return (
+      
+      <Fragment>
+        <div className="center-div">
+          <div className="cognitive_instructions">
+            <h4 className="margin-bottom">
+              Congratulations! You've finished the Cognitive Learning Module.
+            </h4>
+            <h4 className="margin-bottom">Here are some key takeaways:</h4>
+            <div className="flex-boxes">
+            <div>
+              <div className="icon">
+                <EmojiObjects fontSize="large"/>
+              </div>
+                Use proper headings and subheadings to reduce cognitive load
```

Comment:
Not anything to fix, but I love the use of icons here!

Reply:
Thanks!